composer.json

```diff
@@ -1,37 +1,37 @@
-{
-	"name": "microsoft/microsoft-graph",
-	"type": "library",
-	"description": "The Microsoft Graph SDK for PHP",
-	"homepage": "https://developer.microsoft.com/en-us/graph",
-	"license": "MIT",
-	"authors": [
-		{
-			"name": "Microsoft Graph Client Tooling",
-			"email": "graphtooling@service.microsoft.com",
-			"role": "Developer"
-		}
-	],
-	"require": {
-		"php": "^8.0 || ^7.3",
-		"guzzlehttp/guzzle": "^6.0 || ^7.0",
-		"ext-json": "*",
-		"psr/http-message": "^1.0"
-	},
-	"require-dev": {
-		"phpunit/phpunit": "^8.0 || ^9.0",
-		"mikey179/vfsstream": "^1.2",
-		"phpstan/phpstan": "^0.12.90"
-	},
-	"autoload": {
-		"psr-4": {
-			"Beta\\Microsoft\\Graph\\": "src/Beta/Microsoft/Graph/",
-			"Microsoft\\Graph\\": "src/"
-		}
-	},
-	"autoload-dev": {
-		"psr-4": {
-			"Microsoft\\Graph\\Test\\": "tests/Functional/",
-			"Microsoft\\Graph\\Http\\Test\\": "tests/Http/"
-		}
-	}
-}
+{
+	"name": "microsoft/microsoft-graph",
+	"type": "library",
+	"description": "The Microsoft Graph SDK for PHP",
+	"homepage": "https://developer.microsoft.com/en-us/graph",
+	"license": "MIT",
+	"authors": [
+		{
+			"name": "Microsoft Graph Client Tooling",
+			"email": "graphtooling@service.microsoft.com",
+			"role": "Developer"
+		}
+	],
+	"require": {
+		"php": "^8.0 || ^7.3",
+		"guzzlehttp/guzzle": "^6.0 || ^7.0",
+		"ext-json": "*",
+		"psr/http-message": "^1.0"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^8.0 || ^9.0",
+		"mikey179/vfsstream": "^1.2",
+		"phpstan/phpstan": "^0.12.90"
+	},
+	"autoload": {
+		"psr-4": {
+			"Beta\\Microsoft\\Graph\\": "src/Beta/Microsoft/Graph/",
```

Comment:
I don't think we need line 27

Reply:
Caught this late in [another PR](https://github.com/microsoftgraph/msgraph-sdk-php/pull/621/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34). Is this fine?